test/e2e/e2e_suite_test.go

```diff
@@ -28,6 +28,18 @@ const (
 	gateway1Name      = "gw-1"
 	vip1Name          = "vip-1"
 
+	loadBalancerName   = "lb-fe"
+	proxyName          = "proxy"
+	nseName            = "nse-vlan"
+	ipamName           = "ipam"
+	nspName            = "nsp"
+	ipamServiceName    = "ipam-service"
+	nspServiceName     = "nsp-service"
+	roleName           = "meridio-configuration-role"
+	roleBindingName    = "meridio-configuration-role-binding"
+	configmapName      = "meridio-configuration"
+	serviceAccountName = "meridio"
```

Comment:
this is already defined in  "github.com/nordix/meridio-operator/controllers/common"
why not just import this package?